src/OmniSharp.Abstractions/Models/v1/FindSymbols/FindSymbolsRequest.cs

```diff
@@ -7,5 +7,7 @@ public class FindSymbolsRequest : IRequest
     {
         public string Language { get; set; }
         public string Filter { get; set; }
+        public int MinFilterLength { get; set; }
```

Comment:
@DustinCampbell @david-driscoll What's our versioning strategy here? Should this become a "V2" endpoint if we add properties?

Reply:
IMO, adding these properties safe because their are forward and backward compatible and independent from other ones.

Reply:
IMO. this doesn't need to be a new "V2" endpoint because it's just an additional property. However, I wonder if it should be `int?` to signal that it's optional?

Reply:
Your guys call. I see both approaches are used in the current APIs, e.g. https://github.com/OmniSharp/omnisharp-roslyn/blob/6836fadb9c35a88d4695276d14302336a460b841/src/OmniSharp.Abstractions/Models/v1/GotoDefinition/GotoDefinitionRequest.cs#L8 and https://github.com/OmniSharp/omnisharp-roslyn/blob/6836fadb9c35a88d4695276d14302336a460b841/src/OmniSharp.Abstractions/Models/v1/CodeAction/CodeActionRequest.cs#L9

Reply:
... but I personally try to avoid allowing nulls in general and in APIs particularly since they are usually confusing and error prone.

Reply:
@DustinCampbell : I will go with int? then. Thanks.

Reply:
@DustinCampbell, @rchande  : done